### DISCUSSION TITLE
Hitting this error after 1 question - "Please check your firewall rules and network connection then try again. Error Code: ERR_HTTP2_STREAM_CANCEL."

### DISCUSSION THREAD

**Discussion by ramakishorenooji** [community/community#122580] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Repeatedly facing this issue on my second question, 1st question works fine always but as soon as I hit second one this error pops up - 
Please check your firewall rules and network connection then try again. Error Code: ERR_HTTP2_STREAM_CANCEL.

I have checked there is no VPN or other firewall-blocking request in my machine. Please help.

**Comment by queenofcorgis** [comment]
Hi @ramakishorenooji , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details like what the question you're asking is might help someone give you a nudge in the right direction.
